miden-lib/src/lib.rs

```diff
@@ -9,6 +9,8 @@ use assembly::{utils::Deserializable, Library, LibraryNamespace, MaslLibrary, Ve
 mod tests;
 
 pub mod assembler;
+pub mod auth;
```

Comment:
Since we export only a single struct from `auth`, I'd probably just re export the struct rather than the whole module. For example, something like:

```Rust
mod auth;
pub use auth::AuthScheme;
```

Reply:
ok